interface/forms/reviewofs/new.php

```diff
@@ -2,162 +2,203 @@
-<input type=checkbox name='splenectomy'  ><span class=text><?php xl('Splenectomy','e'); ?></span><br>
-
-</td><td valign=top>
-
-<span class=bold><?php xl('Musculoskeletal','e'); ?></span><br>
-<input type=checkbox name='osetoarthritis'  ><span class=text><?php xl('Osetoarthritis','e'); ?></span><br>
-<input type=checkbox name='rheumotoid_arthritis'  ><span class=text><?php xl('Rheumotoid Arthritis','e'); ?></span><br>
-<input type=checkbox name='lupus'  ><span class=text><?php xl('Lupus','e'); ?></span><br>
-<input type=checkbox name='ankylosing_sondlilitis'  ><span class=text><?php xl('Ankylosing Spondlilitis','e'); ?></span><br>
-<input type=checkbox name='swollen_joints'  ><span class=text><?php xl('Swollen Joints','e'); ?></span><br>
-<input type=checkbox name='stiff_joints'  ><span class=text><?php xl('Stiff Joints','e'); ?></span><br>
-<input type=checkbox name='broken_bones'  ><span class=text><?php xl('Broken Bones','e'); ?></span><br>
-<input type=checkbox name='neck_problems'  ><span class=text><?php xl('Neck Problems','e'); ?></span><br>
-<input type=checkbox name='back_problems'  ><span class=text><?php xl('Back Problems','e'); ?></span><br>
-<input type=checkbox name='back_surgery'  ><span class=text><?php xl('Back Surgery','e'); ?></span><br>
-<input type=checkbox name='scoliosis'  ><span class=text><?php xl('Scoliosis','e'); ?></span><br>
-<input type=checkbox name='herniated_disc'  ><span class=text><?php xl('Herniated Disc','e'); ?></span><br>
-<input type=checkbox name='shoulder_problems'  ><span class=text><?php xl('Shoulder Problems','e'); ?></span><br>
-<input type=checkbox name='elbow_problems'  ><span class=text><?php xl('Elbow Problems','e'); ?></span><br>
-<input type=checkbox name='wrist_problems'  ><span class=text><?php xl('Wrist Problems','e'); ?></span><br>
-<input type=checkbox name='hand_problems'  ><span class=text><?php xl('Hand Problems','e'); ?></span><br>
-<input type=checkbox name='hip_problems'  ><span class=text><?php xl('Hip Problems','e'); ?></span><br>
-<input type=checkbox name='knee_problems'  ><span class=text><?php xl('Knee Problems','e'); ?></span><br>
-<input type=checkbox name='ankle_problems'  ><span class=text><?php xl('Ankle Problems','e'); ?></span><br>
-<input type=checkbox name='foot_problems'  ><span class=text><?php xl('Foot Problems','e'); ?></span><br>
-</td>
-</tr>
-</table>
-
-<span class=text><?php xl('Additional Notes: ','e'); ?></span><br><textarea cols=40 rows=8 wrap=virtual name="additional_notes" ></textarea><br>
-<br>
-<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[<?php xl('Save','e');?>]</a>
-<br>
-<a href="<?php echo "$rootdir/patient_file/encounter/$returnurl";?>" class="link"
- onclick="top.restoreSession()">[<?php xl('Don\'t Save','e');?>]</a>
-</form>
+			<div class="col-sm-6 col-md-2 panel panel-primary">
```

Comment:
Organized sections into panels, with clearer headers. Displays all panels in one tab. When multiple panels are open, it displays two columns next to each other to reduce congestion